### PR TITLE
Add 8-bit Level Shifter [TXB0108]

### PR DESCRIPTION
**Pull request by nonas-hunter**
Creating a new component for an 8-bit level shifter to be used on the SD Card CAN Logger PCB.

[Link to Digi-Key](https://www.digikey.com/en/products/detail/texas-instruments/TXB0108PWR/1305699)

### CODE REVIEW

**Comment by jack-greenberg** [comment]
Digikey link is broken @nonas-hunter

**Comment by nonas-hunter** [comment]
Link should be fixed! @jack-greenberg

File: schematic/formula.dcm [hunk 1]
```diff
@@ -1166,6 +1166,12 @@ $CMP TXB0104
 D Voltage level translator, 4-channel, bidirectional, TSSOP-14
 $ENDCMP
 #
+$CMP TXB0108
+D Voltage level translator, 4-channel, bidirectional, TSSOP-14
```
Comment (ljordan51) [inline schematic/formula.dcm:1170]: is it not 8-channel?

Comment (ljordan51) [reply schematic/formula.dcm:1170]: and I think it's TSSOP-20 right?

**Review by ljordan51 — CHANGES_REQUESTED** [review]
I need a little more time to look at the actual symbol, gotta switch computers so just publishing this review for now

**Comment by ljordan51** [comment]
okay everything looks good, just address the inline comments above and then this can be merged

File: schematic/formula.dcm (hunk 1 above)
Comment (nonas-hunter) [reply schematic/formula.dcm:1170]: You are so right -- totally missed that. I'll fix that now.

**Comment by nonas-hunter** [comment]
@ljordan51 -- Changes were just made to the comment! Is there anything else that needs to be addressed?

Also, how did you do that awesome suggestion box?

**Review by ljordan51 — APPROVED** [review]

**Comment by ljordan51** [comment]
haha you hover the mouse just to the right of the line number and a blue plus sign pops up which you can click on to leave an inline comment

**Comment by ljordan51** [comment]
you have to be in the "Files Changed" or "Commits" view where you can see the actual code

**Pull request merged**